### ISSUE TITLE
[cryptolib] HMAC driver TODO list

### ISSUE THREAD

**Issue by ballifatih** [lowRISC/opentitan#23191]
### Description

(this issue is WIP)
- [ ] Consider whether we want to have a timeout value for polling for interrupt signal. HMAC does not use EDN, so we should be able to determine a reasonable timeout value.

**Comment by martin-velay** [comment]
The following is addressed by this PR (once merged): https://github.com/lowRISC/opentitan/pull/24839

"Consider whether we want to have a timeout value for polling for interrupt signal. HMAC does not use EDN, so we should be able to determine a reasonable timeout value."

**Comment by johannheyszl** [comment]
IIUC, we have completed all tasks that are security hardening related, deprioritizing. cc @nasahlpa

**Comment by nasahlpa** [comment]
Yes, the SCA and FI hardening of HMAC is in a good shape. No open security hardening tasks I am aware of.

**Comment by johannheyszl** [comment]
Closing after discussion
